dev/CommonStyles/ToggleButton_themeresources.xaml

```diff
@@ -275,7 +277,7 @@
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ToggleButtonBorderBrushChecked}" />
                                         </ObjectAnimationUsingKeyFrames>
                                         <contract7Present:ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="BackgroundSizing">
-                                            <DiscreteObjectKeyFrame KeyTime="0" Value="OuterBorderEdge" />
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ToggleButtonCheckedStateBackgroundSizing}" />
```

Comment:
@ranjeshj @alexhinz should we add similar resource for all the other states: normal and indeterminate?

Reply:
What do you mean ? I don't see any other hard coded values.

Reply:
I think @beervoley is asking whether we should have a different resource for those states in case users might want to customize them differently.

Reply:
oh I see. I don't think we need that.